Content/Library/tests/MyLib.1.Tests/MyLib.1.Tests.fsproj

```diff
@@ -3,6 +3,7 @@
     <PropertyGroup>
         <OutputType>Exe</OutputType>
         <TargetFramework>net6.0</TargetFramework>
+        <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
```

Comment:
Should tests have this set? Just curious about this.

Reply:
To me, it doesn't make much sense. I would remove the generation of AssemblyInfo.fs using Fake, but I also set the GenerateAssemblyInfo flag to false, since the lack of a version is more honest than the eternal 1.0.0.0. The same goes for the 'build' project. Summing up, it is logical to set the GenerateAssemblyInfo flag to false at the root level, and leave the generation of Fake AssemblyInfo.fs at the src level. If you agree with this, then I will implement it.

Reply:
Yeah that sounds good!

Reply:
> Yeah that sounds good!
Done.